src/common/MemberSelector/MemberSelector.tsx

```diff
@@ -95,9 +83,13 @@ const MemberSelector = <T extends FieldValues>({
           <FormControl>
             <Select
               {...field}
-              value={loginUser.role === UserRole.Member  ? loginUser.name : field.value}
+              value={
+                loginUser?.role === UserRole.Member
+                  ? loginUser?.id.toString()
```

Comment:
This is still there. Either fix it or remove it from this PR and fix independently.

Reply:
Fixed